### DISCUSSION TITLE
meow-use-clipboard

### DISCUSSION THREAD

**Discussion by jgarte** [meow-edit/meow#463] [Q&A]
Hi,

Why is using [meow-use-clipboard](https://github.com/meow-edit/meow/blob/master/CUSTOMIZATIONS.org#meow-use-clipboard) discouraged?

**Comment by DogLooksGood** [comment]
In my opinion it's a good practice to distinguish the system clipboard(with `C-w C-k C-y`) and emacs kill-ring(with meow's commands).
